packages/fast-usdc-contract/src/exos/operator-kit.js

```diff
@@ -25,25 +24,6 @@ const trace = makeTracer('TxOperator');
  * @typedef {Readonly<{ operatorId: string, powers: OperatorPowers }> & {disabled: boolean}} State
  */
 
-const OperatorKitI = {
```

Comment:
it's not clear to me why `OperatorKitI` should be exported.

Reply:
the CLI needs `operator-kit-interface.js` for `INVITATION_MAKERS_DESC`. I could separate the typeguard from the rest of the interface but I don't see the value in that.